src/http_module.cpp

```diff
@@ -536,8 +560,15 @@ ngx_int_t initWorkerProcess(ngx_cycle_t* cycle)
     }
 
     try {
+        auto creds = grpc::InsecureChannelCredentials();
+        if (mcf->pemRootCerts.len) {
```

Comment:
The certificate bundle shouldn't be required to enable SSL, as gRPC will use system's bundle by default.

I think, we should use "https" scheme to enable it (see https://github.com/nginxinc/nginx-otel/commit/668077dbf7cd0e5383c1265c9cecf11f310c7ebf). Most likely, we'll need something like `bool ssl` flag in the `MainConf`.

Reply:
Good catch, thanks.Yes, according to the [docs](https://grpc.github.io/grpc/cpp/structgrpc_1_1_ssl_credentials_options.html) if `pem_root_certs` is empty, the default roots will be used. The default roots can be overridden using the GRPC_DEFAULT_SSL_ROOTS_FILE_PATH environment variable pointing to a file on the file system containing the roots.